src/movies.js

```diff
@@ -1,16 +1,59 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+
+const movies = require('./data');
+
+function getAllDirectors(moviesArray) {
+  let result = moviesArray.map((element) => element.director);
+  return result;
```

Comment:
I would say to you use a more descriptive variable instead of `element` as you did in the others iterations